manager/misc.go

```diff
@@ -266,3 +282,25 @@ func streamLogToFile(logStream io.ReadCloser, path string, errLog io.Writer) {
 		return
 	}
 }
+
+func (m *VolumeManager) InitSupportBundle() (*SupportBundle, error) {
+	if m.sb != nil {
+		if m.sb.State == BundleStateInProgress {
+			return nil, errors.Errorf("longhorn-manager is busy processing another support bundle request")
+		}
+		if m.isPreviousSupportBundleExpired() == false {
+			return nil, errors.Errorf("current support bundle has not expired.")
+		}
+		// Clear the object and references as previous support bundle got expired
+		os.Remove(filepath.Join("/tmp", m.sb.Fname))
+		m.sb = nil
+	}
+	m.sb = NewSupportBundle()
+	fileName, err := m.GenerateSupportBundle()
```

Comment:
```
sb, err := m.GenerateSupportBundle()
if err != nil {
xxx
}
m.sb = sb
```

So if there is an error while processing the request initially, m.sb will still be nil